pkg/minikube/bootstrapper/kubeadm/versions.go

```diff
@@ -260,6 +259,14 @@ var versionSpecificOpts = []VersionedExtraOption{
 		},
 		GreaterThanOrEqual: semver.MustParse("1.11.0-alpha.0"),
 	},
+	{
+		Option: util.ExtraOption{
```

Comment:
Mention this change as part of the PR description.

Reply:
Updated PR description. I had a link to comment https://github.com/kubernetes/minikube/issues/3164#issuecomment-425327164 that explains the reasons, but agree to have it in PR is better.